datanode/sqlstore/migrations/0001_initial.sql

```diff
@@ -1241,7 +1241,7 @@ create table current_liquidity_provisions
     reference text,
     tx_hash bytea not null,
     vega_time timestamp with time zone not null,
-    primary key (id)
+    primary key (id, vega_time)
```

Comment:
This change should be made as part of the 0010_live_liquidity_provisions.sql  migration  (unless I have missed something?)

Reply:
This has been corrected, thanks!